src-site/usage/composition.md

```diff
@@ -105,8 +105,22 @@ By combining [Stacks]({{site.basedir}}/components/stack), [Grids]({{site.basedir
         <div class="ds-field">
           <button class="ds-button" type="submit">Log in</button>
         </div>
-      </form>
-    </div>
+      </form> 
+      <div>
+      <div class="ds-date">
+        <div class="ds-field">
+          <label for="day_0">Day</label>
+          <input type="number" name="day_0" value="12" required="" id="id_date-date_0">
+        </div>
+        <div class="ds-field">
+          <label for="month_1">Month</label>
+          <input type="number" name="month_1" value="12" required="" id="id_date-date_1">
+        </div>
+        <div class="ds-field">
+          <label for="year_2">Year</label>
+          <input type="number" name="year_2" value="2025" required="" id="id_date-date_2">
+        </div>
+      </div>
+      </div>
```

Comment:
@symroe despite tweaking the date picker form in dc_utils, this structure for the DayMonthYear widget in Django doesn't reflect reality and therefore is causing issues with the CSS in this PR. Labels and input are nested inside one common `ds-field` div rather than individually:

`<div class="ds-date">
    <h2>Date</h2>
    
        <small>The date that polls open</small>
    
    
        <span class="ds-error">
            
                <p>year 222222 is out of range</p>
            
        </span>

    
    <div class="ds-field">
            <div class="form-group form-group-day">
                <label for="day_0">Day</label>
                <input type="number" name="date-date_0" value="121" label="Day" required="" id="id_date-date_0">
            </div>
            
            <div class="form-group form-group-month">
                <label for="month_1">Month</label>
                <input type="number" name="date-date_1" value="23" label="Month" required="" id="id_date-date_1">
            </div>
            
            <div class="form-group form-group-year">
                <label for="year_2">Year</label>
                <input type="number" name="date-date_2" value="222222" label="Year" required="" id="id_date-date_2">
            </div>
            </div> 
</div>`

Additionally, the form group divs and classes added by django forms are breaking the css as we have it currently structured. I've fiddled with it quite a bit but I don't see a way to match the structure to what we have shown in this commit; I can edit the CSS to accommodate the structure as shown in this comment, but I wanted to check first I'm not missing something.